Client/src/Store/User/userActionTypes.ts

```diff
@@ -0,0 +1,24 @@
+export const LOGIN = 'LOGIN';
+export const LOGOUT = 'LOGOUT';
+
+export interface userState  {
+    isLogined:boolean,
+    accessToken: string,
+    fullName?: string,
+    email?: string,
+  };   
+
+
+interface SetLogin {
+    type: typeof LOGIN;
+    payload: userState;
```

Comment:
is there a reason for user action payload to hold isLogined as a var?
since you could deduce it from the action type